### PR TITLE
Ahmet js week2 exercises

### PR DESCRIPTION
**Pull request by Ahmet-K**
Ahmet JS Week-2 Exercises

### CODE REVIEW

File: week-2/G-conditionals-2/exercise-2.js [hunk 1]
```diff
@@ -8,8 +8,12 @@
 */
 
 function studentPassed(grade) {
-
-}
+    if (grade < 50) {
+      return "fail";
+    } else {
+      return "passed";
```
Comment (martynamaron) [inline week-2/G-conditionals-2/exercise-2.js]: Well done :)

File: week-2/G-conditionals-2/exercise-3.js [hunk 2]
```diff
@@ -9,8 +9,17 @@
 */
 
 function calculateGrade(mark) {
+    if (mark >= 80) {
+      return "A";
+    } else if(mark>60) {
+      return "B";
+    } else if (mark>50){
```
Comment (martynamaron) [inline week-2/G-conditionals-2/exercise-3.js]: Good! Can you tell me if these conditions have to appear in this particular sequence? What happens when we move them around?

File: week-2/G-conditionals-2/exercise-4.js [hunk 3]
```diff
@@ -9,7 +9,9 @@
 */
 
 function containsCode(sentence) {
-
+  if (sentence.includes("code")){
+    return true;
+  } return false;
```
Comment (martynamaron) [inline week-2/G-conditionals-2/exercise-4.js]: Missing `else` :) But again it's possible to just return one boolean and get rid of `if/else`! :)

File: week-2/L-array-methods-2/exercise2.js [hunk 4]
```diff
@@ -7,7 +7,12 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  var word = str.split(" ");
+  var j = word[0].charAt(0).toUpperCase();
```
Comment (martynamaron) [inline week-2/L-array-methods-2/exercise2.js]: What's `j`? I think it deserves a better name :)

**Comment by martynamaron** [comment]
Well done @Ahmet-K ! Left some comments about `if/else` statements that you can try out, but overall great work :)
Please don't forget the week-2-practice exercises! Thanks 👍

File: week-2-practise/level-1/1-boolean.js [hunk 5]
```diff
@@ -3,7 +3,7 @@
 
 var isHappy = false;
 
-if (isHappy) {
+if (!isHappy) {
```
Comment (martynamaron) [inline week-2-practise/level-1/1-boolean.js]: Yay! Well done :)

File: week-2-practise/level-2/1-boolean.js [hunk 6]
```diff
@@ -4,7 +4,7 @@
 
 var isHappy = "false";
 
-if (isHappy) {
+if (!isHappy) {
```
Comment (martynamaron) [inline week-2-practise/level-2/1-boolean.js]: Not quite - `isHappy` is a string, and usually you'd want a boolean in an `if` statement. How can yu modify the `isHappy` variable then?

File: week-2/C-comparison-operators/exercise.js [hunk 7]
```diff
@@ -7,14 +7,27 @@
...
+}
 
 var personA = "Daniel";
 var personB = "Irina";
-var sameName; // finish this statement
+if (personA==personB){
+  var sameName=true; // finish this statement
+} else {
+  var sameName = false; // finish this statement
+}
```
[the remark this answers is not in the document]
Comment (Alisina92) [reply week-2/C-comparison-operators/exercise.js]: nice work Ahmet

**Pull request closed without merging**